### PR TITLE
Fix some spellings

### PR DESCRIPTION
**Pull request by RoosterDragon**

### CODE REVIEW

File: OpenRA.Game/FieldLoader.cs [hunk 1]
```diff
@@ -704,7 +704,7 @@ public DictionaryFromYamlKeyAttribute()
 		}
 	}
 
-	// mirrors DescriptionAttribute from System.ComponentModel but we dont want to have to use that everywhere.
+	// mirrors DescriptionAttribute from System.ComponentModel but we don't want to have to use that everywhere.
```
Comment (atlimit8) [inline OpenRA.Game/FieldLoader.cs]: Should capitalize mirrors then.

File: OpenRA.Game/FileSystem/ZipFile.cs [hunk 2]
```diff
@@ -34,7 +34,7 @@ public ZipFile(string filename, int priority)
 			this.priority = priority;
 			try
 			{
-				// pull the file into memory, dont keep it open.
+				// pull the file into memory, don't keep it open.
```
Comment (atlimit8) [inline OpenRA.Game/FileSystem/ZipFile.cs]: Should capitalize pull then.

File: OpenRA.Mods.Common/Traits/Air/AttackPlane.cs [hunk 3]
```diff
@@ -39,7 +39,7 @@ public override Activity GetAttackActivity(Actor self, Target newTarget, bool al
 
 		protected override bool CanAttack(Actor self, Target target)
 		{
-			// dont fire while landed or when outside the map
+			// don't fire while landed or when outside the map
```
Comment (atlimit8) [inline OpenRA.Mods.Common/Traits/Air/AttackPlane.cs]: Should capitalize don't then.

File: OpenRA.Mods.Common/Traits/World/SpawnMapActors.cs [hunk 4]
```diff
@@ -29,7 +29,7 @@ public void WorldLoaded(World world, WorldRenderer wr)
 			{
 				var actorReference = new ActorReference(kv.Value.Value, kv.Value.ToDictionary());
 
-				// if there is no real player associated, dont spawn it.
+				// if there is no real player associated, don't spawn it.
```
Comment (atlimit8) [inline OpenRA.Mods.Common/Traits/World/SpawnMapActors.cs]: Should capitalize "if" then.

File: OpenRA.Mods.Common/Traits/Air/AttackPlane.cs (hunk 3 above)
Comment (Phrohdoh) [inline OpenRA.Mods.Common/Traits/Air/AttackPlane.cs]: And end with a full-stop.

**Comment by obrakmann** [comment]
Cause ALL the merge conflicts! :D :+1:

**Comment by pchote** [comment]
:+1:

**Pull request merged**